module1.test.js

```diff
@@ -96,6 +96,8 @@ describe("Answers", function() {
     // tax is 20% of the retail price
     const tax = 0.2;
     const retailPrice = "£10"; 
+    var taxAmount = (20 * retailPrice)/100
+
     solution = exercises.question9(retailPrice, taxAmount);
     expect(solution).toEqual("£12")
   })
```

Comment:
Khaled, if you pull the revised version of the repo down to your local machine, you'll see that this has already been fixed.

To include the changes into your local copy, in the terminal, make sure you are inside the `module-tests` directory, then issue the following commands

The first connects your local copy to the master CYF repo.
```
git remote add cyf https://github.com/CodeYourFuture/module-tests.git
```

This line then allows you to pull updates directly from the CYF master
```
git pull cyf master
```

To check things are as they should be, issue the following command.
```
git remote -v
```
You should get back a response showing 3 lines. One will be showing from the `cyf` repo (fetch), and two from `origin`, which is your own repo. You should see one each, fetch and push.

Reply:
thank you crise